### ISSUE TITLE
Feature Request - Output Files

### ISSUE THREAD

**Issue by puzzlepeaches** [iustin24/chameleon#13]
Chameleon has been great so far, awesome work! Was hoping we could add the ability to output results to a file in JSON/TXT formats for parsing by other tools. Is this something on your roadmap? Ideally the JSON output file would support the `-L` flag when targeting multiple URLs as well. 

Let me know if I can help at all. Thanks!

**Comment by iustin24** [comment]
Hi @puzzlepeaches,

Sorry for taking so long to reach back! Yes, output files are on the roadmap and should get added in the next release.

Thanks for the suggestions!

**Comment by iustin24** [comment]
Hi @puzzlepeaches,

I have added the option to save output to a file using the `-o` flag. ( ca115e8232ecf906d37fb7f3987c2b0d6efdb22f )
You can also use the `-J` flag along side `-o` for json output. ( 0f295e96ce46f220fbc31963a99f0b374046dcd0 ) As you've mentioned, this will also work with the `-L` flag, generating output like so:
```json
{
  "host": "https://google.com",
  "results": [
    {
      "url": "https://google.com/a",
      "status_code": 301,
      "content_length": 221,
      "location": "https://www.google.com/a"
    }
  ]
}
{
  "host": "https://example.com",
  "results": []
}
{
  "host": "https://www.google.com",
  "results": []
}

```

These changes will be included in the next release ( v1.1 ).  If you want to start using them sooner, you can either clone the project and build using `cargo build --release`, or use `cargo install --git https://github.com/iustin24/chameleon`.

Thanks again for the suggestions!
